package-examples/helloworld-kustomize/README.md

```diff
@@ -0,0 +1,22 @@
+helloworld-kustomize
+==================================================
+
+# NAME
+
+  helloworld-kustomize
+
+# SYNOPSIS
+
+  kubectl apply --recursive -f helloworld-kustomize
+
+# Description
+
+a sample where the upstream package contains a kustomize patch that contains a setter.  One can run the pipeline in the package
```

Comment:
`kustomize patch that contains setter`: I am unable to parse this. If we want to showcase an example of `overlay patch`, then we need to use patch function (don't think we are that ready yet) with a patch. Mixing kustomize seems odd here.

Reply:
I agree with that.  I think I want to break out this workstream into two things.  One is making sure that the existing samples "compile" with the kpt `next` the second one is does it make sense to have those samples.

Reply:
ok.